rspace/src/main/scala/coop/rchain/rspace/LMDBStore.scala

```diff
@@ -20,7 +20,8 @@ import scala.collection.immutable.Seq
   *
   * To create an instance, use [[LMDBStore.create]].
   */
-class LMDBStore[C, P, A, K] private (val env: Env[ByteBuffer],
+class LMDBStore[C, P, A, K] private (env: Env[ByteBuffer],
+                                     databaseBath: Path,
```

Comment:
🛀 🛀 🛀 

I assume you meant `databasePath`?

Reply:
Sure :))))